HISTORY.rst

```diff
@@ -0,0 +1,10 @@
+.. :changelog:
```

Comment:
This is basically for Pypi, we can leave just one entry until we release first version to pypi